dbc2val/dbcfeeder.py

```diff
@@ -436,7 +436,7 @@ def _get_kuksa_val_client(command_line_parser: argparse.Namespace,
         client.set_port(config.getint(CONFIG_SECTION_GENERAL, CONFIG_OPTION_PORT))
 
     if config.has_option(CONFIG_SECTION_GENERAL, CONFIG_OPTION_TLS_ENABLED):
-        client.set_tls(config.getboolean(CONFIG_SECTION_GENERAL, CONFIG_OPTION_TLS_ENABLED, fallback=False))
+        client.set_tls(config.getboolean(CONFIG_SECTION_GENERAL, CONFIG_OPTION_TLS_ENABLED, fallback=True))
```

Comment:
I assume it make sense to have the most strict alternative as fallback, so if you for instance write `Ja` or `Nein` it shall be interpreted as `True`